src/App.js

```diff
@@ -1,30 +1,33 @@
 import React from 'react';
 import { Provider } from 'react-redux';
+import { Box, Heading, Stack, Text, Grid } from "@chakra-ui/core";
+import { FaDiceD20 } from 'react-icons/fa';
+
+import './app.css';
 
 import store from './store';
 import Sets from './components/sets/Sets';
 import Cards from './components/cards/Cards';
 import Deck from './components/deck/Deck';
-import './App.css';
 
 const App = () => {
     return (
         <Provider store={store}>
-            <div className="container is-fluid">
-                <h1>
-                    <span className="icon">
-                        <i className="fas fa-dice-d20" aria-hidden="true"></i>
-                    </span>
-                    <span>Deck Builder</span>
-                </h1>
-                <div className="columns">
-                    <div className="column is-one-fifth">
-                        <Sets />
-                        <Deck />
-                    </div>
-                    <div className="column"><Cards/></div>
-                </div>
-            </div>
+            <>
+                <Box bg="gray.600" w="100%" p={4} color="white" borderBottom="1px" borderColor="gray.700">
```

Comment:
I'm torn. It's kinda cool and kinda handy to have little components ready to use and import.
BUT... this box is basically a just a div with a tonne of inline styles.
It's nice to get a look and feel so quickly, but it remains to be seen how manageable all of this will be.